.github/actions/scenarios/database-write-prohibition/mysql/action.yml

```diff
@@ -0,0 +1,57 @@
+name: "Database Write Prohibition Plugin MySQL Test"
+description: "Auto test for mysql write prohibition"
+runs:
+  using: composite
+  steps:
+    - name: entry
+      uses: ./.github/actions/common/entry
+      with:
+        log-dir: ./logs/databse-write-prohibition/mysql
+    - name: start mysql
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y mysql-server
+        sudo systemctl start mysql.service
+        config_file="/etc/mysql/debian.cnf"
+        user=$(sudo awk -F '=' '/user/ {print $2; exit}' "/etc/mysql/debian.cnf" | tr -d '[:space:]')
+        password=$(sudo awk -F '=' '/password/ {print $2; exit}' "/etc/mysql/debian.cnf" | tr -d '[:space:]')
+        sudo mysql -u $user -p$password -e "CREATE DATABASE IF NOT EXISTS test;"
+        echo "MYSQL_USER=$user" >> $GITHUB_ENV
+        echo "MYSQL_PASSWORD=$password" >> $GITHUB_ENV
```

Comment:
need to create env in this repository？

Reply:
To specify the user and password when starting the mysql demo in the next step